### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Pazzo97**
I've added index.htmlfile, style.css file and test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
-# Hello-world
```
Comment (gemmen29) [inline README.md:1]: Kindly use the README template for the [README file](https://github.com/microverseinc/readme-template) with its styling and add the same information you added to that template.

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+Install
+Usage
+Run tests
+Deployment
+Authors
+bust_in_silhouette Author1
+
+GitHub: @pazzo97
+Twitter: @twitterhandle
+LinkedIn: Patrick MUKUNZI
```
Comment (gemmen29) [inline README.md]: Kindly make this information as a hyperlink, you can do that by surrounding username with [] then put the link between ().
```
[username](url)
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Pazzo97, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Kindly rename the branch to a meaningful name, not a `feature-1`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Pazzo97** [comment]
Hello @gemmen29, kindly review my resubmission.

**Review by CollinsTatang — APPROVED** [review]
Hi @Pazzo97 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Pazzo97** [comment]
Thank you @CollinsTatang

**Pull request merged**